cppcache/include/geode/CacheTransactionManager.hpp

```diff
@@ -173,15 +173,15 @@ class _GEODE_EXPORT CacheTransactionManager {
    * @since 3.6.2
    * @see #isSuspended(TransactionId)
    */
-  virtual bool exists(std::shared_ptr<TransactionId> transactionId) = 0;
+  virtual bool exists(TransactionId& transactionId) = 0;
```

Comment:
Could TransactionId be a value type? My thinking is what if someone needed to suspend the tx, keep a copy of the id, maybe pass to another thread, and resume. It was easy to do with shared pointer, harder to do with ref, easy to do as value.

Reply:
I agree that would be nicer.  And originally that was my plan too.  However, I think problem I ran into was that `TransactionId` is an abstraction and actual class being passed is `TXId`.  If you'd like, I can investigate meaning behind this abstraction and whether it can be dropped and then make change to value type accordingly.  Thoughts?

Reply:
Let's investigate separate from this ticket.